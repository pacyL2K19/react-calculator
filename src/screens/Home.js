/* eslint-disable max-len */
import { Box } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({

  marginTop: {
    marginTop: theme.spacing(5),
  },
  textParagraph: {
    fontSize: 18,
    marginTop: theme.spacing(4),

  },
}));

const Home = () => {
  const classes = useStyles();

  return (
    <Box>
      <h1 className={classes.marginTop}>Welcome to our page!</h1>
      <p className={classes.textParagraph}>
        {' '}
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
        aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
        {' '}
      </p>

      <p className={classes.textParagraph}>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
        reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
      </p>
    </Box>
  );
};

export default Home;
