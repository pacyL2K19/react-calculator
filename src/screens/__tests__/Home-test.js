import renderer from 'react-test-renderer';
import Home from '../Home';

describe('Home page', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Home />);
    expect(tree).toMatchSnapshot();
  });
});
