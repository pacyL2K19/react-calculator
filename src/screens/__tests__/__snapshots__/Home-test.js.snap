// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should match the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-3"
>
  <h1
    className="makeStyles-marginTop-1"
  >
    Welcome to our page!
  </h1>
  <p
    className="makeStyles-textParagraph-2"
  >
     Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu 
  </p>
  <p
    className="makeStyles-textParagraph-2"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
  </p>
</div>
`;
